pontoon/base/models.py

```diff
@@ -2153,7 +2175,6 @@ def for_project_locale(
             entities = Entity.objects.filter(pk__in=Entity.objects.filter(Q(*pre_filters)))
         else:
             entities = Entity.objects.all()
-
```

Comment:
Please leave those blank lines, they make the code a lot easier to read. :-)

Reply:
:+1: